doc/about/01_history.rst

```diff
@@ -0,0 +1,37 @@
+History
+=======
+
+The Development of Pastas started in the spring of 2016 at the TU Delft (The Netherlands) and Artesia (The
+Netherlands). Researchers at the TU Delft required a flexible framework that could support future research on time
+series analysis of groundwater data. Consulting company `Artesia <https://www.artesia-water.nl>`_ on the other hand,
+needed a tool that allowed them to perform more complex analyses in a scripted environment. As such, Pastas was
+developed under a strong partnership between academia and private industry, a collaboration that continues until this
```

Comment:
"continues to this day"

Reply:
[4d26710](https://github.com/pastas/pastas/pull/502/commits/4d2671080dfc6660cf34aeb29c457440080e5530)